rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```diff
@@ -1161,6 +1165,37 @@ public String getJobHtml(@HeaderParam("sessionid") String sessionId, @PathParam(
         }
     }
 
+    @Override
+    @GET
+    @Path("jobs/{jobid}/xml")
+    @Produces("application/xml")
+    public InputStream getJobContent(@HeaderParam("sessionid") String sessionId, @PathParam("jobid") String jobId) {
+        try {
+            Scheduler s = checkAccess(sessionId, "/scheduler/jobs/" + jobId + "/livelog");
+
+            //            final Scheduler s = checkAccess(sessionId, "GEt jobs?jobid=" + jobId);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)